core/src/main/java/org/hibernate/ogm/type/impl/TypeTranslatorImpl.java

```diff
@@ -121,4 +135,40 @@ else if ( type instanceof org.hibernate.type.CollectionType ) {
 		}
 		throw log.unableToFindGridType( type.getClass().getName() );
 	}
+
+	/*
+	   We directly pass the converter "database" type as value to the Tuple
```

Comment:
To me it'd feel right if the target type of the converter is used as input for finding the right grid type.

> Don't we rather want the user to have final say on how the type is transformed?

Yes, but that's subject to custom (grid) types, right?

So I think I see attribute converters more as "logical" converters which e.g. change the representation or do some "formatting", whereas grid types do the actual physical conversion into the column type(s) as understood by individual dialects. E.g. one couldn't write several columns via an attribute converter.

Or, in other words, attribute converters are specific to the (business) attribute types of an application. So attribute converters would always emit one of the basic attribute types defined by JPA, whereas grid types make sure that all these potential basic attribute types are mapped into corresponding column types supported by a store.

Or am I actually having a wrong interpretation of the `AttributeConverter` contract?